### ISSUE TITLE
Windows can't recognize USB unit 

### ISSUE THREAD

**Issue by Bnor47** [wagiminator/CH552-MacroPad-plus#5]
Hi
Really like your project!

I'm trying to use it for a MacroPad I bought on Aliexpress. Its almost indentical to the one you have created. Only difference is that the key pins and knob pins are in another order + that it uses USB C. The difference in PINs I have modified in the file you referred to.

![image](https://github.com/wagiminator/CH552-MacroPad-plus/assets/22070123/8890ae1e-a638-4d91-9759-988280a898f7)

My problem is right now that it seems like I have programmed the CH552, but windows can't recognize it as a valid USB. I have struggled with this for some time and have both tried the CH375 driver and the libusb-win32 without luck.

![image](https://github.com/wagiminator/CH552-MacroPad-plus/assets/22070123/00038f07-127a-41e2-96c6-511f41eabef3)
Translation: "USB unit was not recognized"

![image](https://github.com/wagiminator/CH552-MacroPad-plus/assets/22070123/45e91e93-d131-4d72-add6-7b5676d4744d)
Translation: "Unknown USB unit (not valid unit description)"

By using a bit older board version in arduino, I mannaged to get so far:
![image](https://github.com/wagiminator/CH552-MacroPad-plus/assets/22070123/7575fc64-800c-408c-a64d-0d699dcbd79a)

but still windows can't recognize it
Do you have any idea what I am missing here?

**Comment by Bnor47** [comment]
I finally figured it out - not because I understand why, but it works. I had to go all the way back to version 16 of the CH552 board. Now Windows still view it as an unknown USB device; but compared to be before (v18 - 21) it is not blocked from communicating.

There is also a difference within zadiq:
It is now recognized as a HID device
![image](https://github.com/wagiminator/CH552-MacroPad-plus/assets/22070123/5f1fa107-7090-40f9-89fc-63a3584c0a52)
 Before under "Driver", it simply stated "NONE"
